### ISSUE TITLE
Add `uninstall` section on nvidia-drivers package to cleanup /etc links

### ISSUE THREAD

**Issue by geaaru** [macaroni-os/macaroni-funtoo#22]
Considering that the atm the directory `/etc` is protected the link under `/etc/OpenCL/vendors/nvidia.icd` is not updated correctly on upgrade.

Could be correct to add on finalize the remove of this file in the `uninstall` phase.

**Comment by geaaru** [comment]
The possible workaround now is this:
```bash
$> rm -vf /etc/OpenCL/vendors/nvidia.icd 
$> anise miner ri nvidia-drivers
```

**Comment by geaaru** [comment]
Available from release 23.12.
